site/docs/v1/tech/account-management/totp-two-factor.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: doc
+title: Account Management - Add TOTP Two Factor
+description: Managing Accounts - Add TOTP Two Factor
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Add a Two Factor Method - TOTP
+
+The authenticator is the only service enabled by default. To fully enable:
+
+image::account-management/authenticator-add-account.png[Authenicator Add Account, width=1200, role=bottom-cropped]
+
+1. Scan the QR code with your favorite TOTP-based application.
+2. Enter the code given.
+3. Click the `icon:save[type="fas"] Enable` button.
+
+Now you will be presented with recovery codes. Save these in a safe space.
+
+image::account-management/mfa-recovery-codes-crop.png[Recovery Codes, width=1200, role=bottom-cropped]
+
+icon:trophy[] Success! +
+
+Upon the next login, the user will be prompted for a code displayed by an Authenticator App in addition to your password.
```

Comment:
You switch from "the user" to "your password", pick one :) .

" by an Authenticator App" -> " by the Authenticator App"